lib/node_modules/@stdlib/math/base/special/max/benchmark/c/Makefile

```diff
@@ -1,107 +0,0 @@
-#/
```

Comment:
Don't delete this file, revert this commit

Reply:
Resolved